javascript/.gitignore

```diff
@@ -18,6 +18,7 @@
 .env.test.local
 .env.production
 .env.production.local
+notes.txt
```

Comment:
Is there a reason why this was added to the `.gitignore`? If this is something specific to your workflow, then please remove it.